index.rst

```diff
@@ -122,6 +124,29 @@ Indices
 - :ref:`genindex`
 - :ref:`Search <search>`
 
+.. _part-usage:
+
+Use of Rubin Observatory Software
+=================================
+
+All LSST Science Pipelines code is `free software <http://www.gnu.org/philosophy/free-sw.html>`_, licensed under the terms of the `GNU General Public Licence, Version 3 <http://www.gnu.org/copyleft/gpl.html>`_.
+You have the freedom to run, copy, distribute, study, change and improve the software as you see fit, with no strings attached (aside from the GPL requirements when redistributing the code or derivatives).
+Using, modifying or redistributing the LSST Science Pipelines does not make you subject to the LSST Project Publication Policy (`LPM-162 <https://ls.st/lpm-162>`_).
```

Comment:
I recommend moving this first paragraph into a new section called "License" located below "Citing and acknowledging...." and above "More info. In that section, you can also specify that this documentation is license [CC-BY-SA 4.0](https://github.com/lsst/pipelines_lsst_io/blob/main/LICENSE)